pkg/rules/rules.go

```diff
@@ -86,16 +86,26 @@ func filterRules(ruleGroups []*rulespb.RuleGroup, matcherSets [][]*labels.Matche
 		return ruleGroups
 	}
 
+	gc := 0 // Group count.
```

Comment:
This comment is a sign that the variable should be named `groupCount` instead of `gc`.